### PR TITLE
Simplify concat operator description

### PR DESCRIPTION
**Pull request by llewellyn-sl**
Signed-off-by: Llewellyn vd Berg <113503285+llewellyn-sl@users.noreply.github.com>

### CODE REVIEW

File: docs/operator.rst [hunk 1]
```diff
@@ -476,11 +476,10 @@ See also `join`_, `cross`_, `spread`_ and `phase`_.
...
 ------
 
-The ``concat`` operator allows you to `concatenate` the items emitted by two or more channels to a new channel, in such
-a way that the items emitted by the resulting channel are in same order as they were when specified as operator arguments.
+The ``concat`` operator allows you to `concatenate` the items emitted by two or more channels to a new channel. The items emitted by the resulting channel are in the same order as specified in the operator arguments.
 
-In other words it guarantees that given any `n` channels, the concatenation channel emits the items proceeding from the channel `i+1 th`
-only after `all` the items proceeding from the channel `i th` were emitted.
+Given ``n`` number of channels, the concatenation channel emits the items proceeding from the ``(i+1)`` th channel 
+only after `all` the items proceeding from the ``i`` th channel were emitted.
```
Comment (pditommaso) [inline docs/operator.rst:482]:
```suggestion
Given `n` number of channels, the concatenation channel emits the items proceeding from the `i+1 th` channel 
only after `all` the items proceeding from the `i th` channel were emitted.
```

Note single ` wrap the text as italic, and `` as code

**Review by pditommaso — APPROVED** [review]

File: docs/operator.rst (hunk 1 above)
Comment (llewellyn-sl) [reply docs/operator.rst]: All noted. Thanks.
